politeiad/cache/cockroachdb/decred.go

```diff
@@ -203,6 +207,11 @@ func (d *decred) voteResultsMissing(bestBlock uint64) ([]string, []string, error
 		runoff = append(runoff, token)
 	}
 
+	// Update latest best block used to run this procedure
+	d.bbMtx.Lock()
+	d.lastBestBlock = bestBlock
+	d.bbMtx.Unlock()
```

Comment:
Locking code is particularly sensitive since bugs can cause deadlocks so make a getter and setter function for this.  Less repeated code means less chance of bugs.

```
d.bestBlockGet() int64 {}
d.bestBlockSet(int64) {}
```